test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -17,12 +19,31 @@
 #include <seqan3/range/views/take_until.hpp>
 #include <seqan3/range/views/to.hpp>
 
-#include <gtest/gtest.h>
+#include "../iterator_test_template.hpp"
 
 using seqan3::operator""_dna4;
 using seqan3::operator""_dna5;
 using seqan3::operator""_shape;
 
+using iterator_type = std::ranges::iterator_t<decltype(std::declval<seqan3::dna4_vector&>()
+                                                        | seqan3::views::kmer_hash(0b1001_shape))>;
+
+template <>
+struct iterator_fixture<iterator_type> : public ::testing::Test
+{
+    using iterator_tag = std::forward_iterator_tag;
```

Comment:
why is this only forward? Should it not be random access?

Reply:
good catch, I forgot to change it back

Reply:
It actually revealed another bug of the kmer_hash in the `operator[]()` of the iterator. I fixed it in a new commit and altered the Changelog, since I think this semantic change is notable.